modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/categorization/vocabularies/EditGeneralInfo.tsx

```diff
@@ -0,0 +1,198 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayAlert from '@clayui/alert';
+import {LanguagePicker, Provider} from '@clayui/core';
+import ClayForm, {
+	ClayCheckbox,
+	ClayInput,
+	ClaySelectWithOption,
+	ClayToggle,
+} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import {ClayTooltipProvider} from '@clayui/tooltip';
+import {sub} from 'frontend-js-web';
+import React, {useState} from 'react';
+
+import {IVocabulary} from '../types/IVocabulary';
+
+const VISIBILITY_OPTIONS = [
+	{
+		label: Liferay.Language.get('public'),
+		value: 0,
+	},
+	{
+		label: Liferay.Language.get('private'),
+		value: 1,
+	},
+];
+
+export default function EditGeneralInfo({
+	defaultLanguageId,
+	locales,
+	nameInputError,
+	setNameInputError,
+	spritemap,
+	updateVocabulary,
+	vocabulary,
+}: {
+	defaultLanguageId: string;
+	locales: any[];
+	nameInputError: string;
+	setNameInputError: Function;
+	spritemap: string;
+	updateVocabulary: Function;
+	vocabulary: IVocabulary;
+}) {
+	const [isChecked, setIsChecked] = useState<boolean>(true);
+	const [toggled, setToggle] = useState<boolean>(true);
```

Comment:
Do these states get submitted with the data? Maybe these could be named more specifically to the property they are referring to as well.

Reply:
These values will be submitted to the data in a later ticket. When that happens, these state variables will get deleted since they'll just be persisted to the vocabulary state. 
They're just placeholders for now

Reply:
ok! since they're placeholders, those can be updated formally later 👍